libs/tunables/api.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 Olin Electric Motorsports
+ */
+#pragma once
+#include <stdbool.h>
+#include <stdint.h>
+#include <string.h>
+
+#define MAX_NUM_PARAMS (16)
+
+typedef enum { GET, SET, MEASURE } message_type_t;
```

Comment:
This isn't needed here since it's only used in your tunables.c file.

Reply:
Fixed